core/modules/macros/contrastcolour.js

```diff
@@ -29,6 +29,22 @@ exports.params = [
 Run the macro
 */
 exports.run = function(target,fallbackTarget,colourA,colourB) {
```

Comment:
It's simply applying meaningful defaults. All macros in TiddlyWiki depend on core elements / constructs and are specific to at least one application: TiddlyWiki.